.github/workflows/regression-tests.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         bash run-tests.sh -c ${{ matrix.compiler }} -s ${{ matrix.cxx_std }} -d ${{ matrix.stdlib }} -l ${{ matrix.os }}
 
     - name: Run regression tests - Windows version
-      if: matrix.os == 'windows-latest'
+      if: startsWith(matrix.os, 'windows')
```

Comment:
[Explanation] This line was not updated in a change of this workflow. I've noticed that somewhat accidentally looking at a change that should have caused the test run to fail (e.g. in [here](https://github.com/hsutter/cppfront/actions/runs/9504287442/job/26196600829?pr=1103)) while in fact it showed as successful.